resources/compass-runtime-agent/templates/deployment.yaml

```diff
@@ -19,15 +19,36 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "true"
+        container.apparmor.security.beta.kubernetes.io: runtime/default
```

Comment:
In the documentation is states that by default the runtime/default profile is active, see https://kubernetes.io/docs/tutorials/security/apparmor/#profile-reference

Adding the annotation using this profile does not bring any value, it just makes it more transparent. However, we need to maintain that annotations.. also when the feature becomes stable, it will be moved into the container spec which requires migration.

I propose to not add it to every single container as a mandatory action just to have it more transparent that we use the recommended default. @TorstenD-SAP

Reply:
@a-thaler You are right, adding the annotation in that way makes no sense.

Therefor I agree, that we should NOT add it to the deployment spec. I already changed the provided templates (https://github.com/kyma-project/community/pull/738)